.vimrc.bundles

```diff
@@ -174,11 +174,6 @@
             Bundle 'Valloric/YouCompleteMe'
             Bundle 'SirVer/ultisnips'
             Bundle 'honza/vim-snippets'
-        elseif count(g:spf13_bundle_groups, 'neocomplcache')
```

Comment:
Instead of just removing the whole section, we might keep it with a warning message for a while. It would be less surprising but someday we need to get rid of it anyway. What do you think?